src/components/Counter.js

```diff
@@ -20,20 +22,20 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => {this.props.increment()}}>
```

Comment:
You also could have written line 25 as `onClick = {this.props.increment}` without using the callback.